configs/webpack.config.web.prod.ts

```diff
@@ -76,11 +76,11 @@ const configuration: webpack.Configuration = {
     new webpack.EnvironmentPlugin({
       NODE_ENV: 'production',
       APP_ENV: 'web',
-      API_PORT: '5000',
+      RI_APP_PORT: '5000',
       API_PREFIX: '',
       BASE_API_URL: 'api/',
       RESOURCES_BASE_URL:
-        process.env.SERVER_TLS_CERT && process.env.SERVER_TLS_KEY ? 'https://localhost' : 'http://localhost',
+        process.env.RI_SERVER_TLS_CERT && process.env.RI_SERVER_TLS_KEY ? 'https://localhost' : 'http://localhost',
```

Comment:
config.web.prod used for web. I thought that we shouldn't rely on tls and UI should have generic implementation.
@romansergeenkosofteq @zalenskiSofteq Could you confirm that we can remove RESOURCES_BASE_URL env?

Reply:
no problem

Reply:
@nelsonwellsredis Could you please remove then?